src/View.php

```diff
@@ -64,6 +64,8 @@ class View implements jsExpressionable
      * Path to template. If you don't specify full path
      * by starting with '/' then will be prepended by
      * default template path.
+     *
+     * @var Template
```

Comment:
it's actually string first but is replaced with a template. In a way, it combines function of 'defaultTemplate'. If I specify 'string|Template' then code hinting is not working. Something need to be renamed here.